index.js

```diff
@@ -1,156 +1,169 @@
-const request = require("request");
-const fs = require("fs");
-const https = require("https");
-const path = require("path");
-const ProgressBar = require("progress");
-const inquirer = require("inquirer");
-const { firstQuestions, nextQuestions } = require("./questions");
+/* eslint-disable camelcase */
```

Comment:
i see there's a lot of formatting changes in this pr, maybe it would be a good idea to add a vscode or prettier config so we can stick to a consistent format?

Reply:
adding .prettierc in next commit